Source/src/main/java/CyberPunkReas/CovidSimulator/models/Restriction.java

```diff
@@ -35,18 +45,39 @@ public int getTimeRestriction() {
     public void setCurfew(boolean curfew) {
         this.curfew = curfew;
     }
+    
+    public void setMobility_Allow(boolean mobility) {
```

Comment:
use camelCase (start with lower case, separate words with UPPER CASE)

Reply:
(and again, be consistent with the naming style, there are other methods in camelCase)